### ISSUE TITLE
upgrade tensorflow code to NCHW format and fused bn

### ISSUE THREAD

**Issue by suiyuan2009** [ethereon/caffe-tensorflow#139]
in my test NHWC format is 25% slow than NCHW on resnet101, and non fused batch normalization is very slow and uses much more memory than fused bn.
